### PR TITLE
PR: Update Fly.io app from V1 to v2 #385

### PR DESCRIPTION
**Pull request by nelsonic**
This PR updates the `fly.toml` file to `V2` #385

### CODE REVIEW

**Comment by nelsonic** [comment]
@LuchoTurtle please review + merge when you can. 👍

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -70,6 +70,7 @@ jobs:
 
     name: Review App Job
     runs-on: ubuntu-latest
+    # needs: [build]
```
Comment (LuchoTurtle) [inline .github/workflows/pr.yml:73]: Why was this added?

**Review by LuchoTurtle — APPROVED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (nelsonic) [reply .github/workflows/pr.yml:73]: Just to _proactively_ run the review app script regardless of the success/failure of `build` ... can easily be reverted in a future PR. 👌 
Thanks for merging. :shipit:
